src/types/app.ts

```diff
@@ -99,7 +100,8 @@ export const integrationFieldProperties: {
   plex: ['apiKey'],
   pihole: ['apiKey'],
   adGuardHome: ['username', 'password'],
-  homeAssistant: ['apiKey']
+  homeAssistant: ['apiKey'],
+  tdarr: [],
```

Comment:
So there are no required fields to get data from the api?

Reply:
Yeah, Tdarr's API is special

Reply:
No, the API is accessible without any auth. I could not find any options to specify an API key in the Tdarr options either.